src/pages/Main/Main.jsx

```diff
@@ -1,39 +1,36 @@
 import React from "react";
-import styled from "styled-components";
-import Navtrans from "../../Components/Navtrans/Navtrans";
+import MainHeader from "./Components/MainHeader/MainHeader";
+import RecentSearch from "./Components/RecentSearch/RecentSearch";
+import EventBanner from "./Components/EventBanner/EventBanner";
+import SpecialPrice from "./Components/SpecialPrice/SpecialPrice";
+import { useAxios } from "../../Components/Hooks/useAxios";
 
 const Main = (props) => {
+  const { response, error } = useAxios({
+    method: "get",
+    url: "/data/specialPriceData.json",
+  });
+
+  if (response && !error) {
+    return (
+      <>
+        <MainHeader />
+        <RecentSearch />
+        <EventBanner />
+        <SpecialPrice title="국내 특가 항공권" cardData={response.domestic.slice(0, 8)} />
+        <SpecialPrice title="해외 특가 항공권" cardData={response.foreign.slice(0, 8)} />
+        {/* <Footer /> footer가 들어올 자리 */}
+      </>
+    );
```

Comment:
```suggestion
    return (
      <>
        <MainHeader />
        <RecentSearch />
        <EventBanner />
        {response && <SpecialPrice title="국내 특가 항공권" cardData={response.domestic.slice(0, 8)} />}
        {response && <SpecialPrice title="해외 특가 항공권" cardData={response.foreign.slice(0, 8)} />}
        {/* <Footer /> footer가 들어올 자리 */}
      </>
    );
```
매번 if문으로 처리하기 보다 조건부 렌더링을 활용하면 좋을 것 같네요

Reply:
와,,,제가 왜 저렇게 썼을까요.....수정하겠습니다!!